backstory/src/main/webapp/visionUploadPrototype/visionDemo.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Vision API Demo</title>
+
+    <!-- Bootsrap Stylesheet -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
+      integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO"
+      crossorigin="anonymous"
+    />
+
+    <!-- Stylesheets and Fonts API -->
+    <link rel="stylesheet" href="visionDemo.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato" />
+    <script src="visionDemo.js"></script>
+  </head>
+
+  <body onload="fetchBlobstoreUrl(); getAnalyzedImages();">
+    <div class="container-fluid d-flex flex-column justify-content-center my-3">
+      <div class="row header blue-header align-items-center">
+        <div class="col d-flex justify-content-center">
+          <h1>Blobstore and Vision API Demos</h1>
+        </div>
+      </div>
+
+      <div class="row align-items-center mt-2">
+        <form class="col d-flex justify-content-center" id="image-upload" method="post" enctype="multipart/form-data">
+          <label for="image-upload">Select Image:</label>
+          <input type="file" name="image-upload" /><br /><br />
+          <button type="submit">Submit</button>
+        </form>
+      </div>
+    </div>
+
+    <div class="container-fluid d-flex flex-column justify-content-center mt-2" id="image-list"></div>
+
+    <!-- Bootstrap Javascript -->
+    <script
+      src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
```

Comment:
Is there a reason why you add the ``script`` tags at the bottom of your HTML file? I would think it would make more sense to include these tags within the ``head`` tags.

Reply:
Bootstrap documentation specifically says to add the script tags to the bottom (no idea why).

Reply:
I did not know that, but good to know. Thanks for the information.